src/settings/DataImportSettings.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { Settings } from '@folio/stripes/smart-components';
+import { AppIcon, InfoPopover } from '@folio/stripes/components';
+
+import JobProfiles from './JobProfiles';
+import MatchProfiles from './MatchProfiles';
+import ActionProfiles from './ActionProfiles';
+import FieldMappingProfiles from './FieldMappingProfiles';
+import FileExtensions from './FileExtensions';
+
+import css from './DataImportSettings.css';
+
+class DataImportSettings extends Component {
+  sections = [
+    {
+      label: this.renderProfilesLabel(),
+      pages: [
+        {
+          route: 'job-profiles',
+          label: this.generateSettingsLabel('jobProfiles', 'jobProfiles'),
+          component: JobProfiles,
+        },
+        {
+          route: 'match-profiles',
+          label: this.generateSettingsLabel('matchProfiles', 'matchProfiles'),
+          component: MatchProfiles,
+        },
+        {
+          route: 'action-profiles',
+          label: this.generateSettingsLabel('actionProfiles', 'actionProfiles'),
+          component: ActionProfiles,
+        },
+        {
+          route: 'mapping-profiles',
+          label: this.generateSettingsLabel('fieldMappingProfiles', 'fieldMapping'),
+          component: FieldMappingProfiles,
+        },
+      ],
+    },
+    {
+      label: <FormattedMessage id="ui-data-import.settings.other" />,
+      pages: [
+        {
+          route: 'file-extensions',
+          label: <FormattedMessage id="ui-data-import.settings.fileExtensions" />,
+          component: FileExtensions,
+        },
+      ],
+    },
+  ];
+
+  renderProfilesLabel() {
+    return (
+      <div className={css.profilesLabel}>
+        <FormattedMessage id="ui-data-import.settings.profiles" />
+        <InfoPopover
+          allowAnchorClick
+          buttonHref="https://wiki.folio.org/display/FOLIOtips/Creating+and+using+profiles"
```

Comment:
Please move out of here please

Reply:
Personally, I do not see a reason for doing this as this link address is bound to profiles label and in the case when it should be changed this the exact place to edit it.